### ISSUE TITLE
Helm Chart release (1.4.11) does not include enhancement for "fullUrl" for some reason.

### ISSUE THREAD

**Issue by jbehrends** [splunk/splunk-connect-for-kubernetes#704]
The Helm Chart release v1.4.11 says it includes support for using fullUrl.. 
See here:
https://github.com/splunk/splunk-connect-for-kubernetes/releases/tag/1.4.11
> added full_url support for splunk hec by @harshit-splunk in #683

( **NOTE**: The actual value in code is fullUrl not full_url as in the above comment)

I've pulled version 1.4.11 of the chart, and this  enhancement is NOT in the chart..  Chart got published before that feature was merged?

Example:
```
$ helm repo add splunk https://splunk.github.io/splunk-connect-for-kubernetes/
$ helm pull splunk/splunk-connect-for-kubernetes
$ tar -xvf splunk-connect-for-kubernetes-1.4.11.tgz
$ grep "fullUrl" splunk-connect-for-kubernetes/ -r
No results are returned...
```

Can we please re-publish the chart with this enhancement?

Thanks!

**Comment by harshit-splunk** [comment]
Hey @jbehrends, `fullUrl` config is in `value.splunk.hec`

https://github.com/splunk/splunk-connect-for-kubernetes/blob/4b460c24c13bfa8f3d240bcbd29cfb6e25be1076/helm-chart/splunk-connect-for-kubernetes/values.yaml#L22-L23

Which will be used in match section:

https://github.com/splunk/splunk-connect-for-kubernetes/blob/4b460c24c13bfa8f3d240bcbd29cfb6e25be1076/helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/configMap.yaml#L355-L357

**Comment by jbehrends** [comment]
@harshit-splunk I can see it in the code on github.  I'm pointing out that it's not in the published v1. 4.11 chart.  Go download it and look.  It looks like the chat got published before the feature was merged in.... Even though the release notes say it should be there.

**Comment by harshit-splunk** [comment]
Hey @jbehrends, I have looked into this and found helm pull will download from https://github.com/splunk/splunk-connect-for-kubernetes/tree/develop/docs , but `splunk-connect-for-kubernetes-1.4.11.tgz` doesn't have the correct source code. We will update it ASAP. For now, you can download the chart from [release](https://github.com/splunk/splunk-connect-for-kubernetes/releases/tag/1.4.11).
Thanks.

**Comment by harshit-splunk** [comment]
Hey @jbehrends, it has been fixed with latest release 1.4.12
